lib/Runtime/Base/PropertyRecord.cpp

```diff
@@ -134,6 +134,22 @@ namespace Js
         }
     }
 
+#if DBG_DUMP
+	void PropertyRecord::Dump(unsigned indent) const
```

Comment:
nit: space

Reply:
Fixed.  (My emacs config was incorrectly not indenting definitions inside a namespace -- holdover from previous job -- so I was doing it by hand via C-q C-i, also a holdover from the previous job.  Thx for the comment; this motivated me to fix the emacs config to indent within a namespace!)